docs/advanced/api_examples.yml

```diff
@@ -281,6 +281,29 @@
 
 -
   id:
+    "create container with environment"
+
+  doc: |
+    Create a new container.
+
+  request: |
+    POST /v1/configuration/containers HTTP/1.1
+
+    {"host": "%(NODE_0)s",
+     "name": "webserver",
+     "image": "nginx:latest",
+     "environment": {"CONFIG_FILE": "/etc/nginx/nginx.conf", "USER": "webserver"}}
+
+  response: |
+    HTTP/1.1 201 Created
+
+    {"host": "%(NODE_0)s",
+     "name": "webserver",
+     "image": "nginx:latest",
+     "environment": {"CONFIG_FILE": "/etc/nginx/nginx.conf", "USER": "webserver"}}    
```

Comment:
:question: The actual response body from the server doesn't have the line breaks and may have the keys in a slightly different order, but I think that's just a documentation convenience.

Actual response as bytes:

```
{"environment": {"CONFIG_FILE": "/etc/nginx/nginx.conf", "USER": "webserver"}, "host": "192.0.2.2", "name": "nginx", "image": "nginx:latest"}
```